index.js

```diff
@@ -15,4 +15,23 @@ async function start() {
   console.log('You entered: ' + secretNumber);
   // Now try and complete the program.
   process.exit();
-}
+
+  async function start() {
+  while (1) {
+      let answer = await ask ("Are we done? ")
+      answer = answer.ToLowerCase
+      if (answer.includes('yes') {done =true}
+    }
+    console.log('You are done!');
+    process.exit();
+    // Now try and complete the program.
+  
+    async function start() {
+    let done = false; 
+      while (!done) {
+        let answer = await ask ("Are we done? ")
+          answer = answerToLowerCase(answer)
+          if (answer.includes('yes') {done =true}
+        }
+        console.log('You are done!');
+        process.exit();
```

Comment:
We need to make sure that every time a new code block is opened with a curly bracket we're also closing it off. You can check where curly brackets match up in VSCode by highlighting them, and it will automatically highlight the curly bracket that matches up.